[Release]: Privacy policy pages, courseware messaging, and footer update

jgagne opened the pull request:
## What this PR does

Testing: Please test all jump-links and links 

Updates:
- https://courses.gymna.si/privacy-policy

Adds: 
- https://courses.gymna.si/privacy-policy/european-rights
- https://courses.gymna.si/privacy-policy/california-privacy-rights
- https://courses.gymna.si/privacy-policy/privacy-shield-notice

- - -

And:
- Updates footer social icons/links
- Updates courseware course enrollment is closed message

jgagne committed:
Update site footer

- Update social (icon) order

jgagne committed:
Update course enrollment is closed message

- Update layout, alignment and spacing

jgagne committed:
Update course enrollment is closed message

- Update style rules, nesting

jgagne committed:
Update courseware metabar layout

- Remove spacing from level text, for smaller viewports
- Add spacing to course CTA, for smaller viewports
- Update comment

rediris committed:
Add new privacy subpage stubs

jgagne committed:
Update privacy policy

- Update based on https://aquent.com/privacy-policy, effective October 10, 2023

jgagne committed:
Add CCPA and GDPR policies

- Add content from https://aquent.com/privacy-policy/european-rights
- Add content from https://aquent.com/privacy-policy/california-privacy-rights

jgagne committed:
Update page styles

- Update nav
- Update lists
- Add note
- Add table

jgagne committed:
Update CCPA and GDPR policies

- Update link to data category table
- Add fragment for data category table
- Update formatting

jgagne committed:
Update privacy policy

- Remove Privacy Shield Participation section
- Update Privacy Shield Participation link

jgagne committed:
Add Data Privacy Framework Notice

- Add content from https://aquent.com/privacy-policy/privacy-shield-notice

jgagne committed:
Update privacy policies

- Update punctuation
- Update formatting

jgagne committed:
Merge pull request #910 from gymnasium/content/privacy-policy

Privacy policy pages

jgagne committed:
Update privacy policy pages

- Remove stray end a tag
- Update fragment formatting

jgagne committed:
Update privacy policy

- Update formatting

jgagne committed:
Update privacy policy

- Update formatting, again

rediris commented on static-pages/privacy-policy/european-rights/index.html:
```diff
@@ -0,0 +1,268 @@
+<section id="identity-verification">
+  <h2>Identity Verification</h2>
+  <p>We may need to request additional information to verify your identity before we can fulfill your request. Please be aware that the processing of your personal data is necessary for performance of our services. If you request erasure, restriction on processing, or withdraw consent, we may be unable to continue providing the services.</p>
+  <table id="data-category">
```
I wonder if it makes sense to list this at the bottom of the main privacy policy page, since it is referenced both by the Euro and the Cali pages?

jgagne replied:
Let me know if a "Data Category" section on the main privacy policy page, sounds good.

andrewpmiller commented on static-pages/privacy-policy/privacy-shield-notice/index.html:
```diff
@@ -0,0 +1,103 @@
+  <h2>Data Processed</h2>
+  <p>We obtain and process Personal Information from the European Union, United Kingdom, and Switzerland primarily as a data controller. As a data controller we collect and process personal data directly from individuals in the manner described in our Privacy Notice. We process this personal data as needed to run our business, to assist applicants to find assignments or direct hire positions with ...
+  <p>On some occasions Aquent may operate as a data processor, processing data on behalf of and on the instructions of our customers in connection with our services. In these situations, our clients decide what personal data they submit to us, and we process the personal data at our customer’s instructions and pursuant to a contractual agreement. We may access and use this data in response to cus ...
+  <p>For more information on how we use and share personal data please see our <a href="https://thegymnasium.com/privacy-policy/">Privacy Notice</a> and <a href="https://thegymnasium.com/privacy-policy/european-rights">European Rights Notice</a>.</p>
```
@jgagne - please remove the trailing slash on the link to our privacy policy

jgagne committed:
Update privacy policy link

- Remove trailing slash

jgagne committed:
Update privacy policy pages

- Add Data Category section to main privacy policy
- Update CCPA jump-link to Data Category section

rediris commented on static-pages/privacy-policy/california-privacy-rights/index.html:
```diff
@@ -0,0 +1,101 @@
+  <h2>Rights Under the California Consumer Privacy Act (CCPA)</h2>
+  <ul>
+    <li><b>Right to Know:</b> You have a right to be informed about the categories of personal information we collect about you, the purpose for which it was collected, the sources from which it was collected and the categories of third parties with whom we share your personal information.</li>
+    <li><b>We post this information in our Privacy Notice.</b> The personal information we collect can be found in the <b><a href="https://thegymnasium.com/privacy-policy/#information-we-collect">“Information We Collect”</a></b> section, the business or commercial purpose for which it is used can be found in the <b><a href="https://thegymnasium.com/privacy-policy/#use-of-collected-data">“Use of C ...
```
Need to remove the trailing slash at the end of any of our gymnasium URLs.

rediris commented on static-pages/privacy-policy/index.html:
```diff
@@ -70,20 +70,21 @@ <h2>Use of Collected Data</h2>
-  <p>Cookies are small text files that we place on the device you use to access our websites and services. Most internet browsers permit a user to accept, decline or delete cookies. You can control the use of cookies at the individual browser level but if you choose to disable cookies, it may limit your use of certain features or functions on our websites.</p>
+  <p>Aquent uses cookies to help us personalize, secure, and improve the performance of our websites and services. (More Information)</p>
+  <p>Cookies are small text files that we place on the device you use to access our websites and services. Most internet browsers permit a user to accept, decline or delete cookies. You can control the use of cookies at the individual browser level but if you choose to disable cookies, it may limit your use of certain features or functions on our websites.  Click below to adjust your cookies sett ...
   <p><a href="https://aquent.com/privacy-policy/">View more information</a> on our cookies policy and to adjust your cookies settings.</p>
```
This line seems redundant since it points to Aquent's policy and is effectively a clone of this page.

jgagne committed:
Update privacy policy links

- Remove trailing slashes for internal links

rediris reviewed (changes requested):
See notes about trailing slashes in gymnasium urls

jgagne committed:
Update privacy policy

- Update formatting (no typewriters)

rediris reviewed (approved):
Let's just gopher it.
